pkg/chain/gen/celo/contract/BondedECDSAKeepFactory.go

```diff
@@ -60,9 +62,25 @@ func NewBondedECDSAKeepFactory(
 		From: accountKey.Address,
 	}
 
-	transactorOptions := bind.NewKeyedTransactor(
-		accountKey.PrivateKey,
-	)
+	// FIXME Switch to bind.NewKeyedTransactorWithChainID when go-ethereum dep
+	// FIXME bumps beyond 1.9.25.
+	key := accountKey.PrivateKey
+	keyAddress := crypto.PubkeyToAddress(key.PublicKey)
```

Comment:
Are we missing `chainID != nil`  verification in the generated contracts code?

Reply:
Yes, my fault. I did not regenerate contracts after locally updating `keep-common` dependency one commit further. It is fixed now.